README.adoc

```diff
@@ -460,26 +460,26 @@ external_services:
     var_workload: VALUE
 ----
 
-|`JAEGER_SERVICE_NAMESPACE`
-|The Kubernetes namespace that holds the Jaeger service. (default is `istio-system`)
+|`JAEGER_URL`
+|The URL to the Jaeger server. When not set, the URL may be automatically detected from OpenShift or Kubernetes API.
```

Comment:
"The URL to the Jaeger Query service." (As jaeger has several services, "server" perhaps could be ambiguous).

> When not set, the URL may be automatically detected from OpenShift or Kubernetes API.

This is not defined, if we want to replicate same logic as used for Grafana if the URL is not setup, I am fine with that, but then Grafana & Jaeger should follow same logic as basically both are similar external services.

Reply:
Yes, I need to change it